docs/en/ingest-management/fleet/deployment-models.asciidoc

```diff
@@ -1,21 +1,27 @@
-[[fleet-server]]
-= Set up {fleet-server}
+[[deployment-models]]
+= Deployment models
 
-{fleet-server} is required if you plan to use {fleet} for central management.
+There are various models for setting up {agents} to work with {es}.
+The recommended approach is to use {fleet}, a web-based UI in Kibana, to centrally manage all of your {agents} and their policies.
+Using {fleet} requires having an instance of {fleet-server} that acts as the interface between the {fleet} UI and your {agents}.
 
-* For self-managed deployments, you must install and host {fleet-server}
-yourself.
+For an overview of {fleet-server}, including details about how it communicates with {es}, how to ensure high availability, and more, refer to <<fleet-server,What is {fleet-server}?>>.
 
-* **No extra setup is required on {ecloud} unless you want to scale your
-deployment.** {ecloud} runs a hosted version of {integrations-server} that
-includes {fleet-server}.
+The requirements for setting up {fleet-server} differ, depending on your particular deployment model:
 
-ifeval::["{serverless-feature-flag}"=="yes"]
-* On-premises {fleet-server} is not currently available for use in an link:{serverless-docs}[{serverless-full}] environment.
-We recommend using the hosted {fleet-server} that is included and configured automatically in {serverless-short} {observability} and Security projects.
-endif::[]
+{serverless-full}::
+In a link:{serverless-docs}[{serverless-short}] environment, {fleet-server} is configured and managed automatically, beneath the covers. There's nothing that you need to configure.
```

Comment:
@kilfoyle can i rephrase this sentence?  "Fleet Server is offered as a service, it is configured and automatically scaled without the need for any user intervention.

Reply:
Fixed via [58f9fa6](https://github.com/elastic/ingest-docs/pull/636/commits/58f9fa67b0a890eebbfc1ce1cace5518e3ba7507)